OneBusAway_WatchKit_Extension/OBAArrivalsAndDeparturesInterfaceController.m

```diff
@@ -0,0 +1,133 @@
+//
+//  OBAArrivalsAndDeparturesInterfaceController.m
+//  org.onebusaway.iphone
+//
+//  Created by Shannon Young on 4/14/15.
+//  Copyright (c) 2015 OneBusAway. All rights reserved.
+//
+
+#import "OBAArrivalsAndDeparturesInterfaceController.h"
+
+#import "OBAStopWK.h"
+#import "OBAArrivalAndDepartureWK.h"
+
+#import "OBAStopRowController.h"
+
+#define kStopRouteDetailRowType @"Route Detail Row"
+
+@interface OBAArrivalsAndDeparturesInterfaceController ()
+
+@property (nonatomic) NSMutableOrderedSet *arrivalsAndDepartures;
+
+@property (weak, nonatomic) NSURLSessionDataTask *dataTask;
+
+@end
+
+@implementation OBAArrivalsAndDeparturesInterfaceController
+
+- (void)awakeWithContext:(id)context {
+    [super awakeWithContext:context];
+
+    // Setup defaults
+    self.arrivalsAndDepartures = [NSMutableOrderedSet new];
+    self.maxCount = NSIntegerMax;
+    self.minutesBefore = 5;
+    self.minutesAfter = 35;
+}
+
+- (void)didDeactivate {
+    // This method is called when watch view controller is no longer visible
+    [super didDeactivate];
+
+    [self.dataTask cancel];
+    self.dataTask = nil;
+}
+
+- (NSArray *)filteredRouteIds {
+    return nil;
+}
+
+- (void)refreshArrivalsAndDeparturesForStop:(OBAStopWK *)stop {
```

Comment:
I created a separate issue to track modularization of the app.  I agree it makes sense to split the work into two efforts.

Until we go 8.0+ we'll have to settle for shared static library instead of embedded framework :(